Seravo/seravo-plugin
# Update translations of the Seravo plugin

#### What are the main changes in this PR?
This PR includes an update for translations of the Seravo plugin

##### Why are we doing this? Any context or related work?
Issue #286  

#### Manual testing steps?
Change the language of the WordPress environment to Finnish and check the pages of the Seravo Toolbox.

#### Screenshots
![tools_menu](https://user-images.githubusercontent.com/43742796/66828426-ad4ad080-ef59-11e9-8597-7a2b8f6f2517.png)
![change_php_version_postbox](https://user-images.githubusercontent.com/43742796/66828435-b176ee00-ef59-11e9-98ca-4967f158b1fa.png)
![seravo_plugin_updater_postbox](https://user-images.githubusercontent.com/43742796/66828436-b176ee00-ef59-11e9-941d-48b318f6fc3b.png)
![shadows_postbox](https://user-images.githubusercontent.com/43742796/66828437-b20f8480-ef59-11e9-8b83-6b12f4a22ac2.png)
